### ISSUE TITLE
I have adapted this project to LeftWM

### ISSUE THREAD

**Issue by ivohulsman** [i3-gnome/i3-gnome#102]
Hi there,

I have been a short time user of i3-gnome, as I really like the combination of running Gnome with a proper tiling WM on top of that.

Though recently I have experimented with LeftWM and found that I like that better, due to it having more flexible tiling layouts along other things.

I have adapted this project to run LeftWM instead of i3wm on top of Gnome, and so far it runs great!

My question is; do you have a preference on how I should share these changes?

Would you consider adding the LeftWM files to this project? I think this might be inconvenient due to the documentation differences between i3wm and LeftWM.

I could also just fork this and replace all references of i3wm to LeftWM, but I thought I'd ask your preference on this.

Thanks for creating this project!

**Comment by jcstr** [comment]
Hi @ivohulsman. 

Thanks adding support to a new window manager!

I think that the first step to adopting the changes is to see them on the wiki.

Would you like to help us by editing the wiki so you can add them?

Then we will assess to add it in the main branch. What do you think? cc @umoqnier, @muktupavels

**Comment by ivohulsman** [comment]
So sorry for the delay!

I have forked the repository and have pushed the changes to add LeftWM support. I have also updated the README.

I would be happy to update the wiki. I'd also be willing to try and refresh the wiki in general.

Please refer to: [wm-gnome](https://github.com/ivohulsman/wm-gnome)

Note that the name `wm-gnome` is purely a placeholder name to avoid causing confusion for now.

**Comment by muktupavels** [comment]
I don't have objections but it does sound weird to add other window managers to the project that is called `i3-gnome`.

**Comment by ivohulsman** [comment]
@muktupavels I fully agree. I initially wanted to apply as a contributor to this project, but then I moved to `leftwm` so I decided to see how that would work with gnome. Of course it is up to you and the other maintainers to decide what happens to this project, and I respect any choice you make. I have thought of a few different ways to possibly handle this though.

One option would be to rename the project to something more generic, such as `wm-gnome` although that name isn't my personal favorite. That would however allow for more window managers to be added. I would be willing to rewrite/update the documentation to reflect those changes. I do think this approach would be more difficult to maintain and support, especially when other wm's get added.

Another option would be to completely ignore the changes I have made. I would then maintain my fork specifically for `leftwm` and update when your project gets updates.

Any option is of course fine by me, and I would still consider contributing to `i3-gnome` if you decided to go for the second option.

**Comment by muktupavels** [comment]
I am not maintainer... More or less I can only help with `gnome-flashback` related things...
